src/components/SiteHeader/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import { Dropdown, Hyperlink, Icon } from '@edx/paragon';
+
+import SearchBar from '../SearchBar';
+
+import './SiteHeader.scss';
+
+
+const SiteHeader = props => (
+  <div className={`site-header ${props.className}`}>
+    <div>
+      <Link to="/">
+        <img className="site-logo" alt="site logo" src={props.siteLogo} />
+      </Link>
+      <span className="d-none d-md-inline">{props.journalName}</span>
+    </div>
+    <div className="header-actions">
+      {
+        props.isAuthenticated &&
+          <span className="d-none d-sm-inline-block">
+            <SearchBar journalId={props.journalId} history={props.history} />
+          </span>
+      }
+      {
+        props.isAuthenticated ? (
+          <div className="account-info">
+            <Dropdown
+              className="control-btn"
+              title={
+                <div>
+                  <Icon className="fa fa-user" />
+                  <span>Account</span>
+                </div>
+              }
+              buttonType={null}
+              menuItems={
+                [
+                  { label: 'Logout', href: props.logoutPath },
+                  { label: 'My Account', href: props.lmsAccountPath },
```

Comment:
Couple of minor things to match the design:
- Logout option should be last
- "My Account" should be "My Profile" per design
- "My Account" should have an icon